gatsby-source-chec/gatsby/node/onCreateNode.js

```diff
@@ -0,0 +1,44 @@
+const { createRemoteFileNode } = require('gatsby-source-filesystem');
+
+const onCreateNode = async (
+  { node, actions: { createNode }, createNodeId, store, cache },
+  { downloadImageAssets = false }
+) => {
+  if (
+    downloadImageAssets &&
+    node.internal.type === 'ChecProduct' &&
+    node.assets
+  ) {
+    const getImageAssets = async () => {
+      const assetIds = [];
+
+      const fetchImageAssets = node.assets.map(async (asset) => {
+        if (!asset.is_image) return;
+
+        let imageNode;
+
+        try {
+          imageNode = await createRemoteFileNode({
+            url: encodeURI(asset.url),
+            store,
+            cache,
+            createNode,
+            createNodeId,
+          });
+        } catch (e) {
+          console.error('gatsby-source-chec: ERROR', e);
+        }
+
+        assetIds.push(imageNode.id);
```

Comment:
```suggestion
        try {
          assetIds.push((await createRemoteFileNode({
            url: encodeURI(asset.url),
            store,
            cache,
            createNode,
            createNodeId,
          })).id);
        } catch (e) {
          console.error('gatsby-source-chec: ERROR', e);
        }
```

I think what you've got now will try to call `.id` on `undefined`.